From 4e15b60e78d40811dec2c653804df21460cf2509 Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Mon, 4 Jun 2018 17:37:20 +0300
Subject: [PATCH 4375/5242] MLK-17481-4: ASoC: fsl: Don't bail out on
 OFS_PERIPH error

commit  9a096415af3e6a2ec098afdc9094744eae7657ae from
https://source.codeaurora.org/external/imx/linux-imx.git

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit b6fa30e239da2f38cf31508e98a405ef697e233b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_dsp.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/fsl/fsl_dsp.c b/sound/soc/fsl/fsl_dsp.c
index 0a1f2a4..1b4ec55 100644
--- a/sound/soc/fsl/fsl_dsp.c
+++ b/sound/soc/fsl/fsl_dsp.c
@@ -808,8 +808,8 @@ static int fsl_dsp_probe(struct platform_device *pdev)
 	sciErr = sc_misc_set_control(dsp_priv->dsp_ipcHandle, SC_R_DSP,
 				SC_C_OFS_PERIPH, 0x5A);
 	if (sciErr != SC_ERR_NONE) {
-		dev_err(&pdev->dev, "Error system address offset of PERIPH\n");
-		return -EIO;
+		dev_err(&pdev->dev, "Error system address offset of PERIPH %d\n",
+			sciErr);
 	}
 
 	sciErr = sc_misc_set_control(dsp_priv->dsp_ipcHandle, SC_R_DSP,
-- 
1.7.9.5

